modules/dev2plus/hw_01/src/Node/Node.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Node of the tree.
+ *
+ * @public
+ * @constructor
+ * @param {Number|String} data Value of the node.
+ * @param {Node} left Left sibling.
+ * @param {Node} right Right sibling.
+ * @param {Node} parent Parent of the node.
+ */
+class Node {
+  constructor(value, left = null, right = null, parent = null) {
+    this.data = value;
```

Comment:
naming is a bit confusing here, either call it data or value, but make it consistent